Monika After Story/game/script-songs.rpy

```diff
@@ -1157,6 +1157,52 @@ label mas_song_shelter_lyrics:
     m "{i}~Until you're gone~{/i}"
     return
 
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_songs_database,
+            eventlabel="mas_song_cant_help_falling_in_love",
+            category=[store.mas_songs.TYPE_SHORT],
+            prompt="Can't help falling in love",
+            random=True,
+            aff_range=(mas_aff.HAPPY,None)
+        ),
+        code="SNG"
+    )
+
+label mas_song_cant_help_falling_in_love:
+    # these are only here for easy timing manipulation - change to explicit values before merging
+    $ pause_s = 0.3
```

Comment:
I'm not really sure the need to TODO this. Since you're pushing the dialogue yourself you can also just push with these values instead of the lengths here.

Reply:
I'll remove it once I'm sure the timing's good. I've wanted to ask for some feedback on that, actually.